protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -533,6 +540,13 @@ func genStringValidator(g *protogen.GeneratedFile, f *protogen.Field, varName st
 		}
 	}
 
+	// If the text contains a full URL anywhere in it, reject it. This should help us control spam.
+	if rules.GetRejectPartialUrl() {
+		g.P("if ", s12protoPackage.Ident("BreakURLMatcher"), ".MatchString(", varName, ") {")
```

Comment:
I think this regex is not suitable for this task as it will match on any dot followed by a non-whitespace right? That is not a URL and will reject many payloads that are not considered containing a URL. You'll have to use a proper URL matcher regex instead, see my other comment for a suggestion.

Reply:
Absolutely! Could you list out some ACs for this task in the ticket as I am currently not sure what I am trying to achieve as it feels like the goalposts keep moving and I worry I will only end up contributing more incorrect code 🙏